content/300-guides/050-database/500-using-prisma-with-planetscale.mdx

```diff
@@ -157,6 +159,54 @@ npx prisma db push
 
 in the terminal from your project directory.
 
+## How to add in missing relations after Introspection
+
+After introspecting with `npx prisma db pull`, the schema you get may be missing some relations. For example, the following schema is missing a relation between the `User` and `Post` models:
+
+```prisma file=schema.prisma
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  title     String   @db.VarChar(255)
+  content   String?
+  authorId  Int
+
+  @@index([authorId])
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+}
+```
+
+In this case you need to add the relation in manually:
+
+```prisma file=schema.prisma highlight=6,16;add
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  title     String   @db.VarChar(255)
+  content   String?
+  author    User     @relation(fields: [authorId], references: [id])
+  authorId  Int
+
+  @@index([authorId])
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
+}
+```
+
+You then need to run `npx prisma db pull` for a second time. After this, Prisma will keep the manually added fields.
```

Comment:
> You then need to run `npx prisma db pull` for a second time.
Why?

Reply:
I don't know but this is what I was told when writing [this page](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/introspection-typescript-planetscale) (see 'After this, run introspection on your database for a second time'). See Slack comment [here](https://prisma-company.slack.com/archives/C02FNFLDUS3/p1641312453050200?thread_ts=1641305113.044200&cid=C02FNFLDUS3).

Reply:
I think that is just in the context of the tutorial, showing that it now keeps these manual changes on Re-Introspection.

This might actually be an indication that the tutorial is not super clear. After manual changes the user definitely does _not_ have to run `db pull` again, it should change absolutely nothing. It should keep the changes from the moment you add them.

We should probably remove this here and clarify the tutorials (so people do not learn a wrong thing, as you did.)

Reply:
Thanks, I've removed it and have made a note to update this in the getting started guide too.